### PR TITLE
Code Documentation

### PR DESCRIPTION
**Pull request by Naylin15**
Hi, @rafaelje.

This Pull Request in progress adds the code documentation on the all java sources.

Close: https://github.com/flyve-mdm/flyve-mdm-android-inventory/issues/23

### CODE REVIEW

File: inventory/src/main/java/com/flyvemdm/inventory/categories/Battery.java [hunk 1]
```diff
@@ -64,6 +64,11 @@
 	private String health = "";
 	private String technology = "";
 
+	/**
```
Comment (rafaelje) [inline inventory/src/main/java/com/flyvemdm/inventory/categories/Battery.java:67]: please check this indentation

File: inventory/src/main/java/com/flyvemdm/inventory/categories/Inputs.java [hunk 2]
```diff
@@ -52,6 +52,11 @@
 	private static final long serialVersionUID = 4846706700566208666L;
 	private Configuration config;
 
+	/**
```
Comment (rafaelje) [inline inventory/src/main/java/com/flyvemdm/inventory/categories/Inputs.java:55]: please check the indentation

File: inventory/src/main/java/com/flyvemdm/inventory/categories/Inputs.java [hunk 3]
```diff
@@ -63,6 +68,10 @@ public boolean equals(Object obj) {
 		return (!super.equals(obj));
 	}
 
+	/**
```
Comment (rafaelje) [inline inventory/src/main/java/com/flyvemdm/inventory/categories/Inputs.java:71]: please check the indentation

File: inventory/src/main/java/com/flyvemdm/inventory/categories/Networks.java [hunk 4]
```diff
@@ -60,6 +60,11 @@
 	private DhcpInfo dhcp;
 	private WifiInfo wifi;
 
+	/**
```
Comment (rafaelje) [inline inventory/src/main/java/com/flyvemdm/inventory/categories/Networks.java:63]: please check the indentation

File: inventory/src/main/java/com/flyvemdm/inventory/categories/Networks.java [hunk 5]
```diff
@@ -120,34 +129,67 @@ public Networks(Context xCtx) {
 		}
 	}
 
+	/**
+	 * Get the Media Access Control address
+	 * @return string the MAC address
+	 */
 	public String getMacaddr() {
 		return wifi.getMacAddress();
```
Comment (rafaelje) [inline inventory/src/main/java/com/flyvemdm/inventory/categories/Networks.java:137]: please check this double indentation

**Review by rafaelje — DISMISSED** [review]
Please review some comments, great job!

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Pull request merged**